recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "rosco" %}
-{% set version = "2.6.0" %}
+{% set version = "2.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/nrel/rosco/archive/v{{ version }}.tar.gz
-  sha256: ca1c1a6ac53e8220b107accccfb8b5299772c38b7c77cd8d2ba383e9825daeaa
+  sha256: b6a2cda92680cf6a460d194901a2f16c2635710a82788576a6383a3bb189fc83
 
 build:
   number: 0
```

Comment:
@dzalkind Do you know why the `skip:` line is included below? This is typically used to not build for a particular system, but in this case I think you would want to build for all systems. The description of this in the docs is [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skipping-builds).

Reply:
It looks like the bot put it there, it looks like it's not doing anything though because it can't be win and linux.  I'll try getting rid of that.